Thesis.tex

```diff
@@ -58,26 +58,22 @@ \chapter{Introduction}
-moral rights: protect non-economic interests of the author
+Copyright usually only applies to a practical expression and not ideas, procedures or mode of operation. Copyright includes economical and moral rights. The economical rights allow the rights holder to derive financial reward from the use of protected works. The holder of economic rights can authorize or prohibit reproduction, broadcast, public performance, recording, translation and adaptation of protected works. Moral rights protect non-economic interests of the creator like being credited for their work. 
 
-economic rights holder can authorize/prohibit
-reproduction, broadcast, public performance, recording, translation, adaptation
+According to the berne convention copyright is obtained automatically with the act of creation and usually covers a very long time span. There is no registration needed before one can claim and enforce copyright over their own works. Often voluntary registration is available nonetheless to help solve disputes over ownership. Such registrations can help facilitate financial transactions related to a copyright and assignment or transfer of rights to other parties. There exist a number of organizations offering collective management of copyrights, exercising those rights and related transactions on behalf of the rights owner.
 
-according to the berne convention copyright is obtained automatically, without registration
-usually covers a very long timespan
-voluntary registration is often available, can help solve disputes over ownership, facilitate financial transactions, assignment/transfer of rights
+\subsection{Copyright on the internet}
+Many users of the internet have a number of misconceptions about copyright and how it applies to their usage of internet services. The most important misconceptions are outlined in an article by Vorys at al. \cite{Vorys2013}.
 
-collective management of copyright, exercise of copyrights and similar rights by organizations on behalf of rights owner
+Probably the most common misunderstanding is that everything found on the internet can be freely downloaded and used as the basis of own works. However this is nit true in most cases. As mentioned in chapter \ref{sec:CopyDef} copyright is applied automatically to all eligible works and unlicensed reuse is prohibited unless explicitly authorized by the rights owner. The source of the works and whether you payed for i or not -- for instance an iTunes purchase versus a YouTube download -- is not relevant.
 
-\subsection{Copyright on the internet}
-common misconception: everything on the internet can be downloaded and used freely
+Another very common misconception often seen on YouTube is that it suffices to either give credit to the original creator, add a copyright notice or claim so called fair use. Adding a copyright notice or crediting the original author has no legal meaning by itself. roper authorization to use the work is still required.
```

Comment:
Woher hast Du diese misconception? auch von vorys?

Claim fair use: Dies IST eine valide Möglichkeit der Verteidigung. Im Gegenteil ist es eher so, dass die Gerichte im Falle von Fair Use eher für die Angeklagten entscheiden als für die Rechteinhaber, welche das Thema lieber ganz weglassen würden ... aber auch hier brauachts noch ne Quelle

Adding a copyright notice or crediting the original author has no legal meaning by itself - hier vielleicht noch ergänzen, dass dies im Falle von Creative Commons oftmals wirklich genügt, allerdings wird durch CC eine Lizenz erteilt unter der Auflage, dass der Author genannt wird. Hier könnte man präzisieren.

Proper authorization : P fehlt...